report/src/test/java/com/hartwig/actin/report/interpretation/EvaluatedTrialComparatorTest.java

```diff
@@ -54,7 +33,34 @@ private static EvaluatedTrial create(@NotNull String trialId, @Nullable String c
                 .cohort(cohort)
                 .isPotentiallyEligible(false)
                 .isOpen(false)
-                .hasSlotsAvailable(false)
+                .hasSlotsAvailable(hasSlotsAvailable)
                 .build();
     }
+
+    @Test
+    public void canSortEvaluatedTrials() {
+        List<EvaluatedTrial> trials = Arrays.asList(create("trial 7", "cohort 1", true),
+                create("trial 3", "cohort 2 + cohort 3", false, "Event C"),
+                create("trial 3", "cohort 1", false, "Event B"),
+                create("trial 5", "cohort 1", false, "Event D", "Event A"),
+                create("trial 5", "cohort 1", false, "Event C"),
+                create("trial 1", null, false),
+                create("trial 1", "cohort 1", false),
+                create("trial 1", "cohort 2", false),
+                create("trial 2", "cohort 1", false));
+
+        List<EvaluatedTrial> trialList = Lists.newArrayList(trials.get(7),
+                trials.get(4),
+                trials.get(2),
+                trials.get(8),
+                trials.get(1),
+                trials.get(6),
+                trials.get(0),
+                trials.get(3),
+                trials.get(5));
+        trialList.sort(new EvaluatedTrialComparator());
+        Iterator<EvaluatedTrial> trialIterator = trialList.iterator();
+
+        trials.forEach(trial -> assertEquals(trial, trialIterator.next()));
+    }
```

Comment:
Same as above, private static function appears before public

Reply:
The order of functions is fixed in #12.